### PR TITLE
added an rle class and test

### PR DESCRIPTION
**Pull request by hklerer**

### CODE REVIEW

File: src/main/java/klerer/gameoflife/RleParser.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        for (int i = 0; i < rleString.length(); i++) {
+            char c = rleString.charAt(i);
+
+            if (Character.isDigit(c)) {
+                count = count * 10 + (c - '0');
+            } else {
+                if (count == 0) {
+                    count = 1;
+                }
+                switch (c) {
```
[the remark this answers is not in the document]
Comment (hklerer) [reply src/main/java/klerer/gameoflife/RleParser.java:18]: i put a default and it still didn't fix

File: src/test/java/klerer/gameoflife/GameOfLifeTest.java [hunk 2]
```diff
@@ -27,4 +27,24 @@ public void testNextGeneration() {
         // then
         assertArrayEquals(expectedGrid, futureGrid);
     }
+
+    @Test
+    public void testLoadPatternFromRLE() {
```
Comment (andrewoid) [inline src/test/java/klerer/gameoflife/GameOfLifeTest.java]: don't starts test methods with the word "test"

File: src/main/java/klerer/gameoflife/RleParser.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (hklerer) [reply src/main/java/klerer/gameoflife/RleParser.java:18]: I cant figure out how to fix this

[the remark this answers is not in the document]
Comment (hklerer) [reply src/main/java/klerer/gameoflife/RleParser.java:18]: Now it says fixed

**Review by andrewoid — APPROVED** [review]

**Pull request merged**